### PR TITLE
Enhance document for combiner of cpu_vulns_all

### PR DESCRIPTION
**Pull request by xhe123**
Enhance document for combiner of cpu_vulns_all

 - Add descriptions as docstring
 - Add sample output for files

Signed-off-by: Xiaonan He <xhe@redhat.com>

### CODE REVIEW

**Comment by xiangce** [comment]
@xhe123 - is this PR ready for review? You'd better make sure the CI/CD test passes

**Comment by xiangce** [comment]
It fails at:
```
tmp/workspace/insights-core_PR-2274-GGUCXXHGM7Q66Z5CVOQ5KX7QRBDAZROIAXSLOQUMRSGAMQDKKFKQ/insights/combiners/cpu_vulns_all.py:docstring of insights.combiners.cpu_vulns_all.CpuVulnsAll:9:Block quote ends without a blank line; unexpected unindent.
```

File: insights/combiners/cpu_vulns_all.py [hunk 1]
```diff
@@ -24,7 +14,29 @@
...
+    Class to capsulate the parsers of cpu_vulns, which includes all files in
+    ``/sys/devices/system/cpu/vulnerabilities/*``. Files information will be
+    stored in a list of dictionaries, each dictionary is for one file, the
+    dictionary key is the file name, dictionary value is the file content.
+
+    Sample output for files:
+        ``/sys/devices/system/cpu/vulnerabilities/spectre_v1``::
+            Mitigation: Load fences
+
+         ``/sys/devices/system/cpu/vulnerabilities/meltdown``::
```
Comment (xiangce) [inline insights/combiners/cpu_vulns_all.py]: One more space is added. please remove.
And please remove one ":", keep one here is ok.

Comment (xhe123) [reply insights/combiners/cpu_vulns_all.py]: ok, I fixed it as you suggested.

**Pull request merged**